### ISSUE TITLE
[Next 13.4.4] Module parse failed: Identifier 'NextResponse' has already been declared

### ISSUE THREAD

**Issue by ahkhanjani** [clerkinc/javascript#1249]
## Package + Version

- [x] `@clerk/nextjs`

### Dependencies + versions

```json
{  
  "dependencies": {
	"@clerk/nextjs": "4.18.4",
    "next": "13.4.4"
  },
}

```

## Description

Starting the dev server throws this error:

```
./node_modules/.pnpm/@clerk+nextjs@4.18.4_next@13.4.4_react-dom@18.2.0_react@18.2.0/node_modules/@clerk/nextjs/dist/server/authMiddleware.js
Module parse failed: Identifier 'NextResponse' has already been declared (3:6)
| "use strict";
| const NextResponse = require("next/dist/server/web/spec-extension/response").NextResponse;
> const NextResponse = require("next/dist/server/web/spec-extension/response").NextResponse;
| const NextResponse = require("next/dist/server/web/spec-extension/response").NextResponse;
| const NextResponse = require("next/dist/server/web/spec-extension/response").NextResponse;
```

I'm not sure which 13.4.4 canary version started this, but it was fine with canary-3 and not with canary-10. I haven't tried with the between versions. The problem remains with 13.4.4 stable release.

**Comment by nikosdouvlis** [comment]
Hello @ahkhanjani, thanks for the report. 
This is caused by a recent NextJS release (`next@13.4.4`) - downgrading to `13.4.3` for now should unblock you.

That being said, we're aware of the issue and we'll do our best to push out a fix today once #1215 is merged.

**Comment by nikosdouvlis** [comment]
`@clerk/nextjs@4.19.0` should resolve this issues and also greatly improve the bundle size of the edge middleware function. Please let us know if you hit any issues :)
